tests/test_sourmash.py

```diff
@@ -198,7 +250,8 @@ def test_do_basic_compare_using_rna_arg(c):
 def test_do_compare_quiet(c):
     testdata1 = utils.get_test_data('short.fa')
     testdata2 = utils.get_test_data('short2.fa')
-    c.run_sourmash('compute', '-k', '31', testdata1, testdata2)
+    # c.run_sourmash('compute', '-k', '31', testdata1, testdata2)
```

Comment:
ok - this does seem out of place 😆 . would you like me to remove it or do you want me to coach you through removing it?

Reply:
I'd actually like to learn.. :)

Reply:
so, in this case, I think the simplest thing to do is change the code back in a new commit - just remove the comment on the old line, and remove the new line. Once this is merged all that history will be gone.

In the future, there are a variety of techniques you can use such as `git revert` to choose an old commit in the history to revert, if the change is the only thing in that commit; and you can use `git reset HEAD^` to go back to the previous commit (current minus one), if it's the most recent commit. I basically use these three options 99% of the time.